inginious/frontend/plugins/upcoming_tasks/__init__.py

```diff
@@ -97,7 +96,7 @@ def page(self, time_planner):
                 else:
                     del tasks_data[courseid][user_task["taskid"]]
             # Remove courses with no unfinished available tasks with deadline and lti courses
-            if (len(tasks_data[courseid]) == 0) or course.is_lti():
+            if (len(tasks_data[courseid]) == 0): # or course.is_lti(): # removing lti check momentarily
```

Comment:
`is_lti` is a method for a course object, however, here course is a taskset object. 

Why was that check needed and is it still needed ? If yes any idea how I can do that ?